MetaMask/metamask-extension
# Add support for swaps via a direct contract for wrapping and unwrapping

## Explanation
When wrapping / unwrapping tokens on different networks, we will only show 1 quote that will do a swap via a direct contract integration.

## Manual testing steps
  - Try to do a swap from ETH -> WETH. You will notice there are no Advanced Options on the Build Quote page and no link to select from different quotes on the View Quote page, since there will only be 1 quote available for wrapping / unwrapping

## Screenshots
Advanced Options are hidden for wrapping / unwrapping across all supported networks:
![image](https://user-images.githubusercontent.com/80175477/129910592-97b19a09-ae11-477f-a190-13246c052e44.png)

If there is only 1 quote available (which is the case for wrapping / unwrapping), don't show a link to choose from quotes:
![image](https://user-images.githubusercontent.com/80175477/129910807-66255095-4ee8-41ff-81d8-e122e97a6b9d.png)
